projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/index.php

```diff
@@ -781,15 +781,22 @@ function wpcom_site_has_global_styles_in_personal_plan( $blog_id = 0 ) {
 		return false;
 	}
 
-	if ( ! function_exists( '\ExPlat\assign_given_user' ) ) {
+	if ( ! function_exists( '\ExPlat\assign_given_user' ) || ! function_exists( '\WPCOM\Experiments\Internal\get_cached_experiment_by_name' ) ) {
```

Comment:
Do we need this function check since it will run only on WPCOM Simple (we have the IS_WPCOM constant above)? 🤔

Reply:
The code lives in Jetpack, we can't guarantee the function exists.

Reply:
I don't think it would be a problem, but I guess it's better to be safe than sorry. :D